ros/Makefile

```diff
@@ -1,7 +1,14 @@
+ROS_DISTRIBUTIONS := humble jazzy kilted rolling
+USE_LOCAL_SDK := OFF
+EXTRA_DOCKER_ARGS :=
+DOCKER_USER_ARGS :=
 
+ifeq ($(shell uname -s), Linux)
+	DOCKER_USER_ARGS = --build-arg USER_UID=$(shell id -u) --build-arg USER_GID=$(shell id -g)
```

Comment:
👍 

I guess you could also prepend this to EXTRA_DOCKER_ARGS rather than adding a new var, if you want